lib/manageiq/messaging/kafka/queue.rb

```diff
@@ -16,12 +16,13 @@ def publish_messages_impl(messages)
 
         def subscribe_messages_impl(options)
           topic = address(options)
+          session_timeout = options[:session_timeout] if options.key?(:session_timeout)
```

Comment:
this line seems redundant since we will check for `nil` later anyway.

Reply:
:+1: good point, updated